### PR TITLE
Fixed a minor build issue

### PR DESCRIPTION
**Pull request by Miyurz**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1838)

### CODE REVIEW

**Comment by aledbf** [comment]
@Miyurz why this change? the idea of this was to check the `sha1cmd` exists before running any task.

**Comment by chrislovecnm** [comment]
@Miyurz can you please sign the CLA, and you have comments.

File: Makefile [hunk 1]
```diff
@@ -64,10 +64,10 @@ ifdef STATIC_BUILD
   EXTRA_LDFLAGS=-s
 endif
 
-SHASUMCMD := $(shell sha1sum --help 2> /dev/null)
-
 ifndef SHASUMCMD
-	$(error "sha1sum command is not available")
+  SHASUMCMD := $(shell sha1sum --help 2> /dev/null)
+else
+  $(error "sha1sum command is not available")
```
Comment (justinsb) [inline Makefile]: I do think this is the correct fix - not using a tab indentation

File: Makefile [hunk 2]
```diff
@@ -64,10 +64,10 @@ ifdef STATIC_BUILD
   EXTRA_LDFLAGS=-s
 endif
 
-SHASUMCMD := $(shell sha1sum --help 2> /dev/null)
-
 ifndef SHASUMCMD
-	$(error "sha1sum command is not available")
+  SHASUMCMD := $(shell sha1sum --help 2> /dev/null)
```
Comment (justinsb) [inline Makefile]: I'm not sure the logic is right here - we want to error if sha1sum is not available on the path.

**Comment by justinsb** [comment]
I think this was now fixed in #1852 though.  Thank you though @Miyurz and if you can sign the CLA we would love to have you contribute more PRs - sorry this one didn't make it but I broke the build and so we had to get a fix merged :-)

**Pull request closed without merging**